keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/model/UpdateClientScope.kt

```diff
@@ -0,0 +1,30 @@
+package de.klg71.keycloakmigration.keycloakapi.model
+
+data class UpdateClientScope(
+        val name: String,
+        val description: String? = null,
+        val protocol: String,
+        val attributes: Map<String, String>,
+        val protocolMappers: List<ProtocolMapper>? = null)
+
+
+
+fun updateClientScope(name: String, description: String? = null, protocol: String = "openid-connect",
```

Comment:
I didn't find a location where this function is called in an action. This is only a change for the keycloak-api Package is this correct?

Reply:
forgot to delete when I was removing unnecessary update client scope action